src/dev/display/lcd/lv_lcd_generic_mipi.h

```diff
@@ -0,0 +1,224 @@
+#define LV_LCD_MASK_RGB_ORDER							(1 << 3)	/* RGB/BGR Order */
+#define LV_LCD_MASK_LINE_ADDRESS_ORDER					(1 << 4)	/* Line Address Order */
+#define LV_LCD_MASK_PAGE_COLUMN_ORDER					(1 << 5)	/* Page/Column Order */
+#define LV_LCD_MASK_COLUMN_ADDRESS_ORDER				(1 << 6)	/* Column Address Order */
+#define LV_LCD_MASK_PAGE_ADDRESS_ORDER					(1 << 7)	/* Page Address Order */
+
+#define LV_LCD_BIT_FLIP_VERTICAL__NOT_FLIPPED			0
+#define LV_LCD_BIT_FLIP_VERTICAL__FLIPPED				LV_LCD_MASK_FLIP_VERTICAL			/* This bit flips the image shown on the display device top to bottom. No change is made to the frame memory */
+#define LV_LCD_BIT_FLIP_HORIZONTAL__NOT_FLIPPED			0
+#define LV_LCD_BIT_FLIP_HORIZONTAL__FLIPPED				LV_LCD_MASK_FLIP_HORIZONTAL			/* This bit flips the image shown on the display device left to right. No change is made to the frame memory */
+#define LV_LCD_BIT_DATA_LATCH_DATA_ORDER__LTOR			0									/* Display Data Latch Order: LCD Refresh Left to Right */
+#define LV_LCD_BIT_DATA_LATCH_DATA_ORDER__RTOL			LV_LCD_MASK_DATA_LATCH_DATA_ORDER	/* Display Data Latch Order: LCD Refresh Right to Left */
+#define LV_LCD_BIT_RGB_ORDER__RGB						0									/* RGB/BGR Order: RGB */
+#define LV_LCD_BIT_RGB_ORDER__BGR						LV_LCD_MASK_RGB_ORDER				/* RGB/BGR Order: BGR */
+#define LV_LCD_BIT_LINE_ADDRESS_ORDER__TTOB				0									/* Line Address Order: LCD Refresh Top to Bottom */
+#define LV_LCD_BIT_LINE_ADDRESS_ORDER__BTOT				LV_LCD_MASK_LINE_ADDRESS_ORDER		/* Line Address Order: LCD Refresh Botton to Top */
+#define LV_LCD_BIT_PAGE_COLUMN_ORDER__NORMAL			0									/* Page/Column Order: Normal Mode */
+#define LV_LCD_BIT_PAGE_COLUMN_ORDER__REVERSE			LV_LCD_MASK_PAGE_COLUMN_ORDER		/* Page/Column Order: Reverse Mode */
+#define LV_LCD_BIT_COLUMN_ADDRESS_ORDER__LTOR			0									/* Column Address Order: Left to Right */
+#define LV_LCD_BIT_COLUMN_ADDRESS_ORDER__RTOL			LV_LCD_MASK_COLUMN_ADDRESS_ORDER	/* Column Address Order: Right to Left */
+#define LV_LCD_BIT_PAGE_ADDRESS_ORDER__TTOB				0									/* Page Address Order: Top to Bottom */
+#define LV_LCD_BIT_PAGE_ADDRESS_ORDER__BTOT				LV_LCD_MASK_PAGE_ADDRESS_ORDER		/* Page Address Order: Bottom to Top */
+
+/* predefined gamma curves */
+#define LV_LCD_GAMMA_2_2								0x01	/* 2.2 */
+#define LV_LCD_GAMMA_1_8								0x02	/* 1.8 */
+#define LV_LCD_GAMMA_2_5								0x04	/* 2.5 */
+#define LV_LCD_GAMMA_1_0								0x08	/* 1.0 */
+
+/* common pixel formats */
+#define LV_LCD_PIXEL_FORMAT_RGB565						0x55	/* bus: 16 bits, pixel: 16 bits */
+#define LV_LCD_PIXEL_FORMAT_RGB666						0x66	/* bus: 18 bits, pixel: 18 bits */
+
+/* flags for lv_lcd_xxx_create() */
+#define LV_LCD_FLAG_NONE								0x00000000UL
+#define LV_LCD_FLAG_MIRROR_X							0x00000001UL
+#define LV_LCD_FLAG_MIRROR_Y							0x00000002UL
+#define LV_LCD_FLAG_BGR									0x00000008UL
+#define LV_LCD_FLAG_RGB666								0x00000010UL
+
+/* command list */
+#define LV_LCD_CMD_SPECIAL	0xff
```

Comment:
As it's used only as a delay:
```suggestion
#define LV_LCD_CMD_DELAY_MS	0xfe
```